tests/singlerod/short/singlerod_short.omn

```diff
@@ -16,8 +16,8 @@ input "singlerod_short.rtk.xml"
 !################################# PHYSICS ####################################
 
 [PHYSICS=ce ce]
-ce_lib ce_v71
-ce_pole_lib_path "/data/pole_data/pole.h5"
+ce_lib_path "/opt/scale/h5_data/ce_v7.1_endf.h5"
+ce_pole_lib_path "/opt/scale/pole_data/pole.h5"
```

Comment:
Once we finalize an ORNL-generated pole library, we _should_ be able to have it automatically find the library instead of having to specify the absolute path.  Hopefully that will let us get away from having absolute paths specified so the input isn't specific to the location of data on any particular machine.